recipe/meta.yaml

```diff
@@ -1,46 +1,35 @@
-{% set sha256 = "27a849d6205838fb6cc3c1c21cb9800707a661bb21c6ce7fb13e99eb1f8a0c46" %}
-{% set version = "1.57.0" %}
 {% set name = "googleapis-common-protos" %}
+{% set version = "1.57.0" %}
 
 package:
   name: {{ name|lower }}-feedstock
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  sha256: 27a849d6205838fb6cc3c1c21cb9800707a661bb21c6ce7fb13e99eb1f8a0c46
 
 build:
-  number: 0
-  skip: true  # [win and vc<14]
+  noarch: python
+  number: 1
```

Comment:
It seems (at least according to grayskull) that this recipe could be `noarch: python`